utils/validate_versions.py

```diff
@@ -85,9 +85,9 @@ def set_version(
             f.write(new_contents)
 
     if csharp_version is not None:
-        package_version = csharp_version + "-preview"
+        package_version = csharp_version + r"-pre.1"
```

Comment:
Don't think you need the preceding `r` here. Can also convert to an f-string.
```suggestion
        package_version = f"{csharp_version}-pre.1"
```

Reply:
ah, whoops.  this was some half-baked idea about my invalid pickle format error earlier.